assets/css/style.css

```diff
@@ -0,0 +1,19 @@
+@import url("https://fonts.cdnfonts.com/css/cocogoose");
+@import url("https://fonts.cdnfonts.com/css/Lato");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Lato, sans-serif;
+  color: #555;
+  position: relative;
+  overflow-x: hidden;
+}
+
+html {
+  overflow-x: hidden;
+}
```

Comment:
# GREAT JOB MAN
> Everything I write below is not requirements, just my feedback or suggestion for you, ok 😃?

First of all, thank you for such a great video presentation, It really helped me to review your project. Your commits are good actually, do not apologize about that 👌. So cool design and very dry code I would say 😁. Now one thing is you should take some rest and sleep, because as I see your eyes already looking red 👁. 

### **GOOD LUCK 🍀**

Reply:
Thanks for reviewing my project @Rustamxon7 and I really appreciate your kind words. :pray: